firmware/ventilator-controller-stm32/Core/Src/main.cpp

```diff
@@ -641,6 +649,8 @@ int main(void)
     // Independent Sensors
     fdo2.output(hfnc.sensor_vars().po2);
     nonin_oem.output(store.sensor_measurements_raw().spo2, store.sensor_measurements_raw().hr);
+    // *temporary* should be used in the breathing circuit
+    abp.output(hfnc.sensor_vars().p_out);
```

Comment:
We'll need to rename `p_out` to `p_out_above_atm`, as p_out = p_out_above_atm (from ABP) + p_atm (from BMP388)